health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/CustomHorizontalBarChart.js

```
@@ -0,0 +1,539 @@
+            ? { ...value.filters, projectTypeId: projectTypeId }
+            : {
+                ...value?.filters,
+                [filterStack[filterStack.length - 1]["filterKey"]]: filterStack[filterStack.length - 1]?.filterValue,
+                projectTypeId: projectTypeId,
+              },
+        moduleLevel: value?.moduleLevel,
+      },
+      shouldDisplayTargetline
+    );
+    target = Math.ceil(targetResponse?.responseData?.data[0].headerValue) > 0 ? Math.ceil(targetResponse?.responseData?.data[0].headerValue) : 0;
+    targetMessage = "TARGET_DSS_" + targetResponse?.responseData?.data[0].headerName?.replaceAll(" ", "_").toUpperCase();
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)